demisto_sdk/commands/common/tools.py

```diff
@@ -2250,6 +2251,101 @@ def is_test_config_match(
     return False
 
 
+def is_content_item_dependent_in_conf(test_config, file_type):
+    """Check if a line from conf have multiple integration/scripts dependent on the TPB.
```

Comment:
Let's add example here for the 2 cases: one with 2 dependent and second case with only one (regular)